tests/components/sidebar/FAB.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import FAB from '../../../src/components/sidebar/FAB';
+
+describe('Testing <FAB /> component', () => {
+  let wrapper;
+  beforeEach(() => { wrapper = shallow(<FAB />); });
+  it('includes 1 div with class fab', () => {
```

Comment:
You are testing for the presence of nodes in a component. This could get ugly for a component with a larger tree structure. A better way to do this would be using snapshot tests.
Also, I think it would be great to test that the component does not break when rendered.